commands/usernote.js

```diff
@@ -95,31 +95,22 @@ module.exports = {
           }
         });
         if (notes) {
-          if(!(args.all === false)) {
-              return Promise.map(notes.ns, note => formatNote(note, args.subreddit, channel))
-                .map((note, index) => `(${index}) ${note}`)
-                .then(formattedNotes => [`Notes on /u/${args.user} on /r/${args.subreddit}:`].concat(formattedNotes));
-          } else {
-              return Promise.map(notes.ns, note => formatNote(note, args.subreddit, channel))
-                .map((note, index) => { 
-                  if(index<=4) {
-                    return `(${index}) ${note}`;
-                  }
-                })
-                .then(formattedNotes => [`Notes on /u/${args.user} on /r/${args.subreddit}:`].concat(formattedNotes))
-                .then(array => {
-                  var notesRemaining = _.size(notes.ns) >= 5 ? _.size(notes.ns) - 5 : _.size(notes.ns);
-                  return array.concat([`... and ${notesRemaining} more usernotes. Use \`.tag /u/username --all\` to display all.`]);
-                });
-          }
-          
+          return Promise.map(
+            args.all !== false ? notes.ns : notes.ns.slice(0,5),
```

Comment:
I think this can just be `args.all ? notes.ns : notes.ns.slice(0,5)`.

(You might need to remove `all` from the "string" options to get this to work.)